packages/ove-app-alignment/README.md

```diff
@@ -1,23 +1,33 @@
-# OVE monitor alignment app
+# Alignment App
 
-## Purpose
+This app exists to help align the monitors in an OVE installation.
 
-This application exists to help align the monitors in an OVE installation.
+When installing monitors to create a tiled display, the aim is typically to physically align screens with as little space between them as possible. However, in practice it is difficult to achieve a perfect alignment and monitors have bezels with non-zero widths. To compensate for this, adjustments can be made to the coordinates recorded for the layout in the `Clients.json` file.
 
-When installing monitors to create a tiled display, the aim is typically to physically align screens with as little space between them as possible.
-However, in practice it is difficult to achieve a perfect alignment and monitors have bezels with non-zero widths.
-To compensate for this, adjustments can be made to the coordinates recorded for the layout in the``Clients.json`` file.
+## Loading the App
 
-## Use
+The alignment app can be loaded using the OVE APIs:
 
-It enables the display of one of two patterns (a grid of vertical and horizontal lines, or a series of parallel diagonal lines) that span an entire space.
+Windows:
```

Comment:
I suggest swapping the order of the examples, so that the Linux/Mac example (which is both more readable and probably useful to more people) is listed first.